packages/gradient-block/src/components/ColorFlyout.tsx

```diff
@@ -58,7 +58,11 @@ export const ColorFlyout = ({
 
     const setValidColorPosition = (value: string) => {
         const valueAsNumber = parseInt(value);
-        if (valueAsNumber >= 0 && valueAsNumber <= 100) {
+        if (
+            valueAsNumber >= 0 &&
+            valueAsNumber <= 100 &&
+            gradientColors.every((item) => item.position !== valueAsNumber)
```

Comment:
Maybe we could just add the color to the outermost part. E.g. if the valueAsNumber is 103 => add it to 100 and if it is smaller than 0 add it to 0. Atm I can still add a color to higher than 103, but it doesn't place it: https://www.loom.com/share/2d50c89f1b1f4c08bf2bb8be14a6bc04

Reply:
How can you add at position 103? I can't manage to do that